pyexchange/coinone.py

```diff
@@ -245,12 +254,12 @@ def _http_unauthenticated_request(self, method: str, resource: str, body: dict):
 
     def _result(self, result) -> Optional[dict]:
         if not result.ok:
-            raise Exception(f"Coinone API invalid HTTP response: {http_response_summary(result)}")
+            raise RuntimeError(f"Coinone API invalid HTTP response: {http_response_summary(result)}")
 
         try:
             data = result.json()
-        except Exception:
-            raise Exception(f"Coinone API invalid JSON response: {http_response_summary(result)}")
+        except RuntimeError:
```

Comment:
You're more likely to get a `JSONDecodeError` error here, which subclasses `ValueError`.  Recommend catching `ValueError`.

Reply:
+1